fedimint-sqlite/Cargo.toml

```diff
@@ -15,6 +15,7 @@ sqlx = { version = "0.7.0-alpha.2", default-features = false, features = ["runti
 fedimint-core ={ path = "../fedimint-core" }
 rand = "0.8"
 tracing = "0.1.37"
+hex = "0.4.3"
```

Comment:
We tried to get rid of the `hex` dependency some time ago and instead used the impl inside `bitcoin_hashes`. Not sure if worth the hassle though … @mxz42

Reply:
Not sure either, but [at some point](https://github.com/rust-bitcoin/rust-bitcoin/pull/1573) we can depend on their `hex` crate as we depend on `bitcoin_hashes` anyway. 
we also use the [`format_hex`](https://docs.rs/bitcoin_hashes/0.11.0/bitcoin_hashes/hex/fn.format_hex.html) macro in a few places.